generate_ruby_client.sh

```diff
@@ -0,0 +1,5 @@
+java -jar public/doc/openapi-generator-cli.jar generate \
```

Comment:
I considered 3 paths: 
- /bin
- /public/doc, where is yaml file located
- gem root